### ISSUE TITLE
State/county/postal town issue for Great Britain, Guernsey, Isle of Man, Jersey, Northern Ireland

### ISSUE THREAD

**Issue by gyroscopes** [zippopotamus/zippopotamus#30]
So in Great Britain, Guernsey, Isle of Man, Jersey, Northern Ireland you don't have a STATE.
The closest match would be a county. 

However it would be very good idea to return the county, the postal town.
The Royal Mail likes address is this format http://www.royalmail.com/personal/help-and-support/How-do-I-address-my-mail-correctly

Which means the county is usually left out. Of course people using this service may want to check the address and county.

Basically the data that is returned for these countries needs to be returned in a format that we can use. At the moment it is not very usable.
